### ISSUE TITLE
Remove placeholder example

### ISSUE THREAD

**Issue by joelanman** [alphagov/accessible-autocomplete#555]
The placeholder example as it stands is not accessible - the text is low contrast and some screen readers don't read out placeholder

https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/

Maybe replace with an example that uses Hint instead?

**Comment by Jordan-Hall** [comment]
@joelanman wouldn't that just be a normal span below? In the example it still uses placeholders. It's just not meant to show key information about the field
